returnn/datasets/hdf.py

```diff
@@ -270,14 +278,16 @@ def get_data(self, seq_idx, key):
     end_pos = self.file_seq_start[file_idx][real_file_seq_idx + 1]
 
     if key == "data":
+      assert 'inputs' in fin, "'data' key is reserved for 'inputs' in the HDF file, but 'inputs' not found in HDF file."
```

Comment:
Maybe we might want to allow `"data"` as a possible entry in `targets/`?

Reply:
Yes, would probably be good. So just
```
if key == "data" and 'targets/data/data' not in fin:
```
here?

To actually create an HDF with data as target we would need to change the HDFDatasetWriter.

Reply:
Yea. I'm not sure we need that logic now, but we should keep that in mind.